source/about-mongosync.txt

```diff
@@ -157,6 +157,28 @@ If the reverse sync is successful, ``mongosync`` enters the ``RUNNING`` state.
 The synchronization continues in the reverse direction from the original 
 sync job. 
 
+.. _mongosync-considerations:
+
+Considerations for Continuous Sync
+----------------------------------
+
+Consider the following when using ``mongosync`` for continuous synchronization:
+
+- For the destination cluster to have an up to date, consistent view of the 
+  source cluster’s data, ``mongosync`` must :ref:`commit <c2c-api-commit>` . 
+  If the source cluster shuts down before ``mongosync`` can commit, such as in 
+  a disaster scenario, the destination cluster might not have a consistent 
+  snapshot of the source data, and transactions might not appear atomically on 
```

Comment:
Question for @rohshar , I would like to confirm my understanding of what happens in a hypothetical scenario where mongosync is mid-sync and the source connection severs, assuming the user performs the steps in the disaster recovery tutorial to undo all the temporary changes that mongosync does on the destination.

Aside from torn transactions, are there any other consequences to call out other than just having potentially stale data? E.g., if a user could guarantee they didn't use transactions, what else could go wrong if their source crashed prior to commit and they manually tried to undo mongosync's work?